handout/testing/services/alt-http-mocking.md

```diff
@@ -0,0 +1,83 @@
+# Alternative HTTP Mocking Strategy
```

Comment:
@SethDavenport @winkerVSbecks possible blog-post idea 'non-insane HTTP testing in Angular 2' ? :P